src/main/java/org/folio/rest/camunda/controller/WorkflowController.java

```diff
@@ -28,13 +28,13 @@ public WorkflowController(CamundaApiService camundaApiService) {
   @PostMapping(value = {"/activate", "/activate/"}, produces = { MediaType.APPLICATION_JSON_VALUE })
   public Workflow activateWorkflow(@RequestBody Workflow workflow, @TenantHeader String tenant)
       throws WorkflowAlreadyActiveException, ScriptTaskDeserializeCodeFailure {
-    log.debug("Activating Workflow: {}", workflow == null ? null : workflow.getId());
+    log.info("Activating Workflow: {}", workflow == null ? null : workflow.getId());
```

Comment:
```suggestion
    log.debug("Activating Workflow: {}", workflow == null ? null : workflow.getId());
```

Should be `debug`.

Reply:
These are not debugging logs and should be aware of each workflow activation/deactivation as normal operating logs. The number of times these are actually logged is insignificant and can be info for useful activity logging.

Reply:
The previous code has them as `debug`.

I do not see how changing that is in the scope of this ticket.
I think excessive logging concerns should be a convesation for a different issue other than the Kafka.

Reply:
There was no logging provided for easily verifying normal operation. Previously with debug might be incorrect as hiding critical information about an active workflow that can indicate a scheduled workflow will be running. Perhaps we can have an issue for defining logging.

Reply:
Updated.